nth-prime.json

```diff
@@ -0,0 +1,34 @@
+{
+  "#": [
+    "The tests that expect 'false' should be implemented to raise",
+    "an error, or indicate a failure. Implement this in a way that",
+    "makes sense for your language."
+  ],
+  "cases": [
+    {
+       "description": "first",
```

Comment:
Consistency in descriptions: either add 'prime' here or remove it for `sixth` and `big` 
My preference is add it.